jbpm-flow/src/main/java/org/jbpm/ruleflow/instance/RuleFlowProcessInstance.java

```diff
@@ -46,4 +48,32 @@ public void internalStart(String trigger) {
     	    .forEach(austoStartNode -> signalEvent(austoStartNode.getName(), null));
     }
 
+
+    private void trigger(RuleFlowProcessInstance wfp, String nodeId) {
```

Comment:
What's the point of passing wfp here, isn't this always this?

Reply:
sure